### PR TITLE
add create-tomochain-masternode doc

### LINKED ISSUE #133: Document create-tomochain-masternode
**Estimated time:** --h (irrelevant)

---------------

**Why:**
New tool coming up to run masternodes for beginners to intermediate level. Needs to be documented

**What:**
- [x] readme
- [x] docs tutorial

### PR DESCRIPTION
**Pull request by etienne-napoleone**
closes #133

### CODE REVIEW

File: docs/masternode/create-tomochain-masternode.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
+We created this tool to help beginner to intermediate masternode owners.
```
Comment (phamvancam2104) [inline docs/masternode/create-tomochain-masternode.md]: I dont think using `we` is good. Why not `This tool is used for helping masternode owners from beginners to intermediate level of skills to facilitate their masternodes' operations`

File: docs/masternode/create-tomochain-masternode.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+It acts like a terminal wizard which helps you generate a working configuration for docker-compose to run your masternode.
```
Comment (phamvancam2104) [inline docs/masternode/create-tomochain-masternode.md]: `create-tomochain-masternode` acts like ... (you have never mentioned the name of the tool :)

Comment (phamvancam2104) [inline docs/masternode/create-tomochain-masternode.md]: helps `masternode owners` ... to run a masternode

File: docs/masternode/create-tomochain-masternode.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+It includes the following features:
```
Comment (phamvancam2104) [inline docs/masternode/create-tomochain-masternode.md]: `It` => `the tool`?

File: docs/masternode/create-tomochain-masternode.md [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+
+- working out of the box docker-compose configuration
+- auto-restart your node on failure
```
Comment (phamvancam2104) [inline docs/masternode/create-tomochain-masternode.md]: `auto-restarting`?

File: docs/masternode/create-tomochain-masternode.md [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+  - Docker-compose
+
+### Installation of Docker CE
+
+To install Docker, first update the apt package index.
+```
+sudo apt update
+```
+
+Then Install packages to allow apt to use a repository over HTTPS.
```
Comment (phamvancam2104) [inline docs/masternode/create-tomochain-masternode.md]: `Install` => `install`

File: docs/masternode/create-tomochain-masternode.md [hunk 6]
```diff
@@ -0,0 +1,201 @@
...
+```
+apt-key fingerprint 0EBFCD88
+```
+
+Set up the stable Docker repository.
+```
+sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+```
+
+Update the apt package index. Then install the latest version of Docker CE.
```
Comment (phamvancam2104) [inline docs/masternode/create-tomochain-masternode.md]: replace dot `.` with comma `,`

File: docs/masternode/create-tomochain-masternode.md [hunk 7]
```diff
@@ -0,0 +1,201 @@
...
+```
+sudo chmod +x /usr/local/bin/docker-compose
+```
+
+Optional: install bash completion.
+```
+sudo curl -L https://raw.githubusercontent.com/docker/compose/1.23.2/contrib/completion/bash/docker-compose -o /etc/bash_completion.d/docker-compose
+```
+
+Verify that docker-compose is installed correctly by running the version command.
```
Comment (phamvancam2104) [inline docs/masternode/create-tomochain-masternode.md]: `correctly installed`?

File: docs/masternode/create-tomochain-masternode.md [hunk 8]
```diff
@@ -0,0 +1,201 @@
...
+  The storage location for your node data (chaindata).
+  Either `docker volume` if you want to use the default docker volume location, or `host directory` if you want to define specific location on your filesystem (usefull when extending storage).
+
+- **Chaindata**:
+  The name of the docker volume that will be used or the path to the directory to mount, depending on your answer to the last question.
+
+- **Expose RPC**:
+  If you want to expose or not port `8545`.
+  It is the RPC api to your node.
+  It should be only exposed if you have a specific reason to access the TomoChain JSON-RPC Protocol.
```
Comment (phamvancam2104) [inline docs/masternode/create-tomochain-masternode.md]: `should only be`

File: docs/masternode/create-tomochain-masternode.md [hunk 9]
```diff
@@ -0,0 +1,201 @@
...
+Once finished, you will get a folder named after your masternode (in our case "mymasternode") with two files.
+
+- `.env`
+  Which contains the configuration generated from your answers to the question.
+
+- `docker-compose.yml`
+  Which tells docker-compose how and which container run for your node.
+  Your specific configuration will be read from the `.env` file.
+
+Now that we generated the correct initial configuration for docker-compose, we just need to start our node.
```
Comment (phamvancam2104) [inline docs/masternode/create-tomochain-masternode.md]: `We have generated`

File: docs/masternode/create-tomochain-masternode.md (hunk 7 above)
Comment (etienne-napoleone) [reply docs/masternode/create-tomochain-masternode.md]: :fr:

**Comment by etienne-napoleone** [comment]
Thanks for the review @phamvancam2104 !

**Pull request merged**